app/main.py

```diff
@@ -32,13 +46,13 @@ def main() -> None:
     ]
 
     # run the programs
-    service.run_program(wake_up_program)
-    service.run_program(sleep_program)
+    await run_sequence(service.run_program(wake_up_program))
+    await run_parallel(service.run_program(sleep_program))
```

Comment:
![image](https://github.com/mate-academy/py-asyncio-iot/assets/103201010/79c13d7d-8e50-4743-bb9d-94c756f9b857)
fix that